v-next/hardhat-ignition-core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nomicfoundation/ignition-core",
-  "version": "3.0.0-next.21",
```

Comment:
Why the bump to 24? `v-next` is on 21

Reply:
Whoops! Changed for verdaccio testing and forgot to change back before committing. I'll fix that.